tutorials/Application.getInfo.md

```diff
@@ -1,4 +1,35 @@
-Retrieves information about the application.
+Retrieves information about the application.  If the application was not launched from a manifest, the call will return the closest parent application `manifest` and `manifestUrl`.  `initialOptions` shows the parameters used when launched programmatically, or the `startup_app` options if launched from manifest.  The `parentUuid` will be the uuid of the immediate parent (if applicable).
+
+#### response
+```
+{
+    initialOptions: {
+        name: "myApp",
+        url: "https://openfin.co",
+        uuid: "myApp",
+        autoShow: true
+    },
+    launchMode: 'fin-protocol' || 'fins-protocol' || 'shortcut' || 'command-line' || 'adapter' || 'other' || a process name (ex: 'openfin.exe'),
```

Comment:
'other' || a process name (ex: 'openfin.exe')    ?

Reply:
@wenjunche   what do you mean?